lib/xhr.js

```diff
@@ -182,11 +183,24 @@ function enableXHROverH2(XMLHttpRequest, configuration) {
                 this.__dispatchEvent(new ProgressEvent('progress'));
                 break;
             case XMLHttpRequest.DONE:
-                redefine(this, 'getResponseHeader', function(header) {
-                    return options.response.headers[header.toLowerCase()];
+                redefine(this, 'getResponseHeader', function (header) {
+                    var responseHeaders = options.response.headers;
+                    var trailers = options.response.trailers;
+                    for (var trailer in trailers) {
```

Comment:
Isn't there a way to include all the trailer headers with a single call, instead of a loop ?